components/jupyter-web-app/backend/kubeflow_jupyter/common/api.py

```diff
@@ -98,6 +98,19 @@ def list_notebooks(namespace):
     )
 
 
+@auth.needs_authorization("list", "core", "v1", "events")
```

Comment:
@fediazgon it will need to be:
`@auth.needs_authorization("list", "", "v1", "events")` (without the core)
just like with pvcs. When written with `core` it will fail the authorization checks. 

I realized that I've written `"core"` instead of `""` in some other api calls like the `list_namespaces` and `get_secret`. Could you also update these ones as well in a follow up commit?

Reply:
Done